pkgs/tools/networking/openssh/default.nix

```diff
@@ -9,7 +9,7 @@
 , autoreconfHook
 , etcDir ? null
 , hpnSupport ? false
-, withKerberos ? true
+, withKerberos ? (stdenv.hostPlatform.system != "armv6l-linux")
```

Comment:
I think this needs a comment "why", e.g. "upstream doesn't support it currently", "it doesn't make sense to", "we can't build it yet", "some runtime failure".

Also, here's another `openssh` / kerberos related PR, not sure how relevant to this: #107606

Reply:
Yeah I merged the other one. Maybe this isn't needed anymore.

Reply:
https://github.com/NixOS/nixpkgs/pull/107606 fixed the build, also for armv6l. Thanks for the pointer!

Reply:
Nice!